### PR TITLE
added logic for pipeline flow - git operations

### PR DESCRIPTION
**Pull request by yzainee**

### CODE REVIEW

**Comment by yzainee** [comment]
[test]

**Comment by yzainee** [comment]
[test]

**Comment by yzainee** [comment]
[test]

**Comment by yzainee** [comment]
[test]

**Comment by yzainee** [comment]
[test]

File: openshift/template.yaml [hunk 1]
```diff
@@ -136,6 +136,14 @@ objects:
             value: "6162"
           - name: F8A_SERVER_SERVICE_HOST
             value: bayesian-api
+          - name: GIT_COMMITTER_NAME
+            value: "osio"
+          - name: GIT_COMMITTER_EMAIL
+            value: "osio@sample.com"
```
[the remark this answers is not in the document]
Comment (yzainee) [reply openshift/template.yaml]: I just wanted to use some sample email id

File: openshift/template.yaml [hunk 2]
```diff
@@ -136,6 +136,14 @@ objects:
             value: "6162"
           - name: F8A_SERVER_SERVICE_HOST
             value: bayesian-api
+          - name: GIT_COMMITTER_NAME
+            value: "osio"
```
[the remark this answers is not in the document]
Comment (yzainee) [reply openshift/template.yaml]: ok

File: f8a_worker/workers/git_operations.py [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+            self.log.exception("Failed to call the backbone.")
+        os.system("rm -rf " + path)
+
+
+class DependencyFinder:
+    """Implementation of methods to find dependencies from manifest file."""
+
+    def __init__(self):
+        """Init function."""
+        return None
```
[the remark this answers is not in the document]
Comment (yzainee) [reply f8a_worker/workers/git_operations.py]: editor was giving a suggestion that class is without init function so i added it

[the remark this answers is not in the document]
Comment (miteshvp) [reply f8a_worker/workers/git_operations.py]: You can use pass instead of return None

**Comment by miteshvp** [comment]
@yzainee - one minor comment, can be taken up in the future as well.

**Pull request merged**